docs/release-notes/RELEASE-NOTES-0.13.0.md

```diff
@@ -2,6 +2,8 @@
 
 [This release][this-release] is focused on TODOFILLMEIN.
 It:
+* Change the cryptographic signing to use Curve25519. This means all old node/signing keys are not useable
+  anymore (`key/key.pub` is replaced by `sign_key/sign_key.pub`).
```

Comment:
Lower in this document, 'even if you keep your key pair" suggests it may be possible to keep keypair, that may be misleading.

Reply:
Yes, I saw that comment. I think temporarily removing it for this release would make sense.